lib/assertions.js

```diff
@@ -798,7 +798,22 @@ module.exports = function (expect) {
 
     expect.addAssertion('<Error> to [exhaustively] satisfy <Error>', function (expect, subject, value) {
```

Comment:
Actually it would be nice to add some tests for `<Error> to exhaustively satisfy <Error>`